seeed-voicecard

```diff
@@ -144,6 +175,15 @@ fi
 
 alsactl restore
 
-#Fore 3.5mm ('headphone') jack
-amixer cset numid=3 1 
+#Force 3.5mm ('headphone') jack
+#   The Raspberry Pi 4, released on 24th Jun 2019, has two HDMI ports,
+#   and can drive two displays with audios for two users simultaneously,
+#   in a "multiseat" configuration. The earlier single virtual ALSA
+#   option for re-directing audio playback between headphone jack and HDMI
+#   via a 'Routing' mixer setting was turned off eventually to allow
+#   simultaneous usage of all 3 playback devices.
+if aplay -l | grep -q "bcm2835 ALSA"; then
+    amixer cset numid=3 1 || true
+fi
 
```

Comment:
This is very wierd - you seems to have branched off a long while ago and now including changes which have happened upstream meanwhile. Should you rebase first?

Reply:
Oops sorry. This was intended to be merged with my master branch. Was working with Git windows application that has raised this pulled request. Sorry for the inconvinience.